tests/test_transforms/test_inference/conftest.py

```diff
@@ -28,6 +28,72 @@ def regular_ts(random_seed) -> TSDataset:
     return tsds
 
 
+@pytest.fixture
+def regular_ts_one_month(random_seed) -> TSDataset:
+    periods = 100
+    df_1 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df_1["segment"] = "segment_1"
+    df_1["target"] = np.random.uniform(10, 20, size=periods)
+
+    df_2 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df_2["segment"] = "segment_2"
+    df_2["target"] = np.random.uniform(-15, 5, size=periods)
+
+    df_3 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df_3["segment"] = "segment_3"
+    df_3["target"] = np.random.uniform(-5, 5, size=periods)
+
+    df = pd.concat([df_1, df_2, df_3]).reset_index(drop=True)
+    df = TSDataset.to_dataset(df)
+    tsds = TSDataset(df, freq="M")
+
+    return tsds
+
+
+@pytest.fixture
+def regular_ts_start_month(random_seed) -> TSDataset:
+    periods = 200
+    df_1 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
```

Comment:
You must set freq in `pd.date_range` function, because default freq="D". May be you should use `etna.datasets.generate_ar_df` function for dataset generation.

Reply:
This dataset generator is not needed at all, so I will delete it